tests/templates/test_embeddings/test_iqp_emb.py

```diff
@@ -42,9 +42,32 @@ def test_expansion(self, n_wires, expected_names, expected_wires):
         op = qml.IQPEmbedding(features, wires=range(n_wires))
         tape = op.expand()
 
+        j = 0
         for i, gate in enumerate(tape.operations):
             assert gate.name == expected_names[i]
             assert gate.wires.labels == tuple(expected_wires[i])
+            if gate.name == "RZ":
+                assert gate.data[0] == features[j]
+                j += 1
+
+    @pytest.mark.parametrize("n_wires, expected_names, expected_wires", QUEUES)
+    def test_expansion_broadcasted(self, n_wires, expected_names, expected_wires):
```

Comment:
Is this a problem that the test name is similar to the one above (in a different module though)?

Reply:
I do not think so, no. The collection in pytest seems to differentiate where the function came from :) 
I'd assume we would be in major trouble, if not :sweat_smile: